src/cli/argument-parser.ts

```diff
@@ -173,6 +173,7 @@ export default class CLIArgumentParser {
             .option('--disable-multiple-windows', 'disable multiple windows mode')
             .option('--disable-http2', 'disable the HTTP/2 proxy backend and force the proxy to use only HTTP/1.1 requests')
             .option('--cache', 'cache web assets between test runs')
+            .option('--baseUrl <url>', 'set the base url for all tests')
```

Comment:
You should use `kebab-case` for naming CLI options. It automatically converted to `camelCase` for the config.